packages/builder/src/components/backend/TableNavigator/popovers/EditTablePopover.svelte

```diff
@@ -66,7 +66,9 @@
   <div slot="control" class="icon">
     <Icon s hoverable name="MoreSmallList" />
   </div>
-  <MenuItem icon="Edit" on:click={editorModal.show}>Edit</MenuItem>
+  {#if internal}
+    <MenuItem icon="Edit" on:click={editorModal.show}>Edit</MenuItem>
+  {/if}
   <MenuItem icon="Delete" on:click={showDeleteModal}>Delete</MenuItem>
```

Comment:
Should we also hide the delete button? In my testing there, trying to delete a datasource plus table does nothing but says it's been deleted. It actually sends a request to a URL ending in `/undefined` which the server 500's on, but the front end says success.... I guess the correct thing to do would be to remove it from the list (like un-import it) but we aren't actually deleting it.

Reply:
Deletion should work, that is implemented for data source plus tables, I can look at the specific scenario that caused a 500 - the `/undefined` is because of the way our delete endpoint works, its `/:id/:rev` and plus tables don't have a `_rev` so they just send undefined, but it does still work (usually).

Reply:
^ Lol i'll revert the commit I just pushed

Reply:
@aptkingston @mike12345567 Is this okay now then?

Reply:
Yeap looks good to me!

Reply:
Fire away!